themes/default/content/registry/packages/kubernetes/how-to-guides/choosing-the-right-helm-resource-for-your-use-case.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Choosing the Right Helm Resource For Your Use Case"
+meta_desc: Learn which Helm Resource is right for your use case
+aliases: ["/docs/reference/tutorials/kubernetes/chart-vs-release/"]
+layout: how-to-guide
+---
+
+The Kubernetes provider and SDK has supported a means to deploy [Helm Charts](https://helm.sh/) since [2018]({{< relref "/blog/using-helm-and-pulumi-to-define-cloud-native-infrastructure-as-code">}}) through the [`Helm V2 Chart`]({{< relref "/registry/packages/kubernetes/api-docs/helm/v2/chart">}}), and later, [`Helm V3 Chart`]({{< relref "/registry/packages/kubernetes/api-docs/helm/v3/chart">}}) resources. These resources simulated Helm installation by retrieving the templates for underlying resources from the chart and installing them on the target Kubernetes environment directly.
+
+In September 2021 we announced the **public preview** of a new [`Helm Release`]({{< relref "/registry/packages/kubernetes/api-docs/helm/v3/release">}}) which adds an additional option to the mix for Pulumi's Kubernetes users. As of [v3.15.0](https://github.com/pulumi/pulumi-kubernetes/releases/tag/v3.15.0) of the Pulumi Kubernetes SDK and Provider, this resource is now **Generally Available**.
+
+This guide discusses in some detail each of these options and provides a framework to determine using the right resource for your use case.
+
+## Helm Chart Resources
+
+The [`Helm V2 Chart`]({{< relref "/registry/packages/kubernetes/api-docs/helm/v2/chart">}}), and [`Helm V3 Chart`]({{< relref "/registry/packages/kubernetes/api-docs/helm/v3/chart">}}) resources render the templates from your chart and applies them directly, which means all provisioning happens client-side using your Kubernetes authentication setup without needing a server-side component such as `Tiller`. These resources are implemented as [`Component Resources`]({{< relref "/docs/intro/concepts/resources/components/" >}}) which unlocks a number of benefits for Pulumi users, e.g.:
+
+### Benefits
+
+1. Visibility into all resources encapsulated by the Chart in Pulumi's state, allowing users the ability to directly query properties of individual resources.
+2. The above also allows Pulumi's Policy-as-Code framework - [`CrossGuard`]({{< relref "/docs/guides/crossguard/" >}}) to enforce policies on all resources installed by Helm charts
+3. Ability to leverage transformations to programmatically manipulate resources installed by Helm charts in any of the Pulumi supported programming languages
```

Comment:
Would be nice to add a link to more info on transformations

Reply:
Done.